src/Induction/WellFounded.agda

```diff
@@ -68,22 +68,21 @@ module _ {_≈_ : Rel A ℓ₁} {_<_ : Rel A ℓ₂} where
 module Some {_<_ : Rel A r} {ℓ} where
 
   wfRecBuilder : SubsetRecursorBuilder (Acc _<_) (WfRec _<_ {ℓ = ℓ})
-  wfRecBuilder P f x (acc rs) = λ y y<x →
-    f y (wfRecBuilder P f y (rs y y<x))
+  wfRecBuilder P f x (acc rs) = λ y<x → f _ (wfRecBuilder P f _ (rs y<x))
 
   wfRec : SubsetRecursor (Acc _<_) (WfRec _<_)
   wfRec = subsetBuild wfRecBuilder
 
   unfold-wfRec : (P : Pred A ℓ) (f : WfRec _<_ P ⊆′ P) {x : A} (q : Acc _<_ x) →
-                 wfRec P f x q ≡ f x (λ y y<x → wfRec P f y (acc-inverse q y y<x))
+                 wfRec P f x q ≡ f x λ y<x → wfRec P f _ (acc-inverse q y<x)
   unfold-wfRec P f (acc rs) = refl
 
 
 ------------------------------------------------------------------------
 -- Well-founded induction for all elements, assuming they are all
 -- accessible:
 
-module All {_<_ : Rel A r} (wf : WellFounded _<_) ℓ where
+module All {_<_ : Rel A r} (wf : WellFounded _<_) {ℓ} where
```

Comment:
This also needs to go in the CHANGELOG

Reply:
I reverted the change instead. My nitpicking takes lower priority.